build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -60,15 +62,25 @@
                             SetupRestoreTestsInputs;"
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
-    <Exec Command="$(DotnetUnderTest) restore $(TestDirectory) --fallbacksource $(TestPackagesDir)" WorkingDirectory="$(TestDirectory)" />
+    <DotNetRestore  FallbackSource="$(TestPackagesDir)"
+                    ToolPath="$(DotnetUnderTest)" 
+                    WorkingDirectory="$(TestDirectory)" />
   </Target>
  
   <Target Name="CleanTestProjectsBinObj" 
           DependsOnTargets="SetupRestoreTestsInputs;" 
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
-    <Delete Files="$(TestDirectory)/**/bin/**/*.*')" />
-    <Delete Files="$(TestDirectory)/**/obj/**/*.*')" />
+    <ItemGroup>
+      <FilesToDelete Include="$(TestDirectory)**/bin/**/*.*" />
```

Comment:
Finding folders requires finding files first, then deleting their parents, as far as I've seen. This is just less code and accomplishes nearly the same...